AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/capture/DefaultCameraCaptureSource.swift

```diff
@@ -146,6 +146,13 @@ import UIKit
         }
         session.commitConfiguration()
         updateOrientation()
+        // `session.startRunning()` should be called after `session.commitConfiguration()` is complete.
+        // However, occacsionally `commitConfiguration()` runs asynchronously, so when `startRunning()`
+        // is called, `commitConfiguration()` is still in progress and the state is still `uncommited`.
+        // This can be reproduced by repeatedly switching or toggling the camera using the RN demo.
+        // Adding a 100ms delay ensures that `session.commitConfiguration()` is complete before calling
+        // `session.startRunning()`.
+        Thread.sleep(forTimeInterval: 0.1)
```

Comment:
Just curious, is this recommended officially? How do we know 100ms is sufficient?

Reply:
There is no official recommendation. I tried 0.05, 0.1 and 0.2, all work on my phone, so I picked 0.1. QA also verified with various devices, no issues found.